### ISSUE TITLE
Upgrade velero to get "ignore volumes of non-running pods"

### ISSUE THREAD

**Issue by Xartos** [elastisys/compliantkubernetes-apps#838]
**What Chart should be upgraded?**

/e.g., dexidp/dex 2.15/

Velero 1.8.1 (?) 1.9.0 (?)

The PR https://github.com/vmware-tanzu/velero/pull/4584 has now been merged and since we want to have the feature we should upgrade as soon as the next version is released.

**Why should this Chart be upgraded?**

- [ ] We need to perform a security upgrade.
- [ ] We need it for a new feature: *which feature (link to blocked issue)*.
- [ ] We need to keep up to prevent future issues.
- [x] Other: *Write the reason here* We want the "ignore volumes of non-running pods" feature

### Definition of done

* Have a velero version that includes the "ignore volumes of non-running pods" feature

**Comment by OlleLarsson** [comment]
The linked PR has been merged @Xartos.
Give this issue some love in the next planning meeting, i'd like this fixed as it will reduce the number of alerts generated

**Comment by Xartos** [comment]
Yes, it has been merged for some time, we've only waited for it to be a part of a release. But the 1.9 release seems to include it :tada:
